nni/tools/nnictl/launcher_utils.py

```diff
@@ -128,5 +128,3 @@ def validate_all_content(experiment_config, config_path):
 
     if 'maxExecDuration' in experiment_config:
         experiment_config['maxExecDuration'] = parse_time(experiment_config['maxExecDuration'])
-    if 'maxTrialDuration' in experiment_config:
-        experiment_config['maxTrialDuration'] = parse_time(experiment_config['maxTrialDuration'])
```

Comment:
why remove "maxTrialDuration" here?

Reply:
It use parse_time() to convert time format to seconds here, but it convert the number in nniManager again. Only need to keep one convert.

Reply:
https://github.com/microsoft/nni/blob/master/ts/nni_manager/core/nnimanager.ts#L435

Reply:
seems "maxExecDuration" is also converted in nnimanager. line 425

Reply:
It is "maxExperimentDuration", not "maxExecDuration" here. The two values seems duplicated.